client/src/components/Ratings/Ratings.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { getReviewsByItem, getReviewsMetaByItem } from '../../../.././utils/apiCalls.js';
```

Comment:
i think these imports can be deleted here since you imported them on the server side, unless you plan on doing something with them here, then disregard. Just trying to provide a decent code review and you're forcing me to be nit-picky.